integration-tests/env/test_environment.go

```diff
@@ -112,7 +113,8 @@ func (b *TestEnvironmentBuilder) Build(t *testing.T) (cldf.Environment, error) {
 	// Only fund wallets when using my-local-ton.
 	if b.Type == CTF || b.Type == LOCAL {
 		for _, chain := range env.BlockChains.TonChains() {
-			testutils.FundWallets(t, chain.Client, []*address.Address{chain.WalletAddress}, []tlb.Coins{tlb.MustFromTON(DefaultFundAmountTon)})
+			ferr := testutils.FundWallets(t, chain.Client, []*address.Address{chain.WalletAddress}, []tlb.Coins{tlb.MustFromTON(DefaultFundAmountTon)})
```

Comment:
any reason not to call it `err`? Calling it different makes it look like we are avoding overwriting `err` for some reason.

Reply:
It's to avoid error variable shadowing, I'm sure in some cases it's ok though. I'll clean up once I got CLDF bump merged in core

Reply:
You won’t shadow the variable as long as you assign to err using `=` instead of `:=`.
If the function returns more than just an error, you can pre-declare the variable to avoid shadowing.
```suggestion
			err = testutils.FundWallets(t, chain.Client, []*address.Address{chain.WalletAddress}, []tlb.Coins{tlb.MustFromTON(DefaultFundAmountTon)})
```

Reply:
Thanks for the clarification. I’ll merge this first to unblock other PRs and handle it in a follow-up